ballerina/metadata_types.bal

```diff
@@ -31,18 +31,33 @@ public type SQLMetadata record {|
     map<JoinMetadata> joinMetadata?;
 |};
 
+
```

Comment:
```suggestion
```

Reply:
updated in [4346c39](https://github.com/ballerina-platform/module-ballerina-persist/pull/201/commits/4346c3959a465aeb05de0872b94ea1056a715a9a)